### PR TITLE
Daily card iterator

### PR DESCRIPTION
**Pull request by GreysonElkins**
###### What's this PR do?
* creates html skeleton for each daily data card
* uses an iterator to populate their values - `populateDailyData()`
* eliminates the need for massive HTML blocks in `page-manipulation` or large amounts of `querySelectors`
* creates a parent class to share methods
###### Where should the reviewer start?
* `page-manipulation` line 3
* `Fitlit.js` and each class
###### What are the relevant tickets?
#27 (three `.join/.map`s are left)
#35 
###### Questions:
* What should we name our parent helper class? `Fitlit` is maybe not the best.

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -1,60 +1,62 @@
-class Activity {
-  constructor(activityData) {
-    this.activityData = activityData
+import Fitlit from "./Fitlit";
+
+class Activity extends Fitlit {
+  constructor(data) {
+    super(data);
```
Comment (JoshSevy) [inline src/Activity.js:5]: Cool, I see where you are adding the inheritance. Looks good.

File: src/Hydration.js [hunk 2]
```diff
@@ -1,31 +1,33 @@
-class Hydration {
-  constructor(hydrationData) {
-    this.hydrationData = hydrationData;
+import Fitlit from './Fitlit';
```
Comment (JoshSevy) [inline src/Hydration.js:1]: Cool so it looks like we are easily to inherit using the super class to access data smoothly. Looks so much better.

File: src/page-manipulation.js [hunk 3]
```diff
@@ -1,39 +1,17 @@
...
-//     }).join("");
-// }
-
-// function makeHydrationHTML(id, hydrationInfo, userStorage, method) {
-//   return method.map(
-//       (drinkData) =>
-//         `<li class="historical-list-listItem">On ${drinkData}oz</li>`
-//     ).join("");
-// }
+function populateDailyData(card, repo, userId, date) {
```
Comment (JoshSevy) [inline src/page-manipulation.js:3]: Looks good, cleaner and easier to read.

File: src/Activity.js [hunk 4]
```diff
@@ -1,60 +1,62 @@
-class Activity {
-  constructor(activityData) {
-    this.activityData = activityData
+import Fitlit from "./Fitlit";
+
+class Activity extends Fitlit {
```
Comment (DougieDev) [inline src/Activity.js:3]: I saw where you mentioned looking for a better name for this in your PR. While we'll want to rename this later, I think for the purposes of having a working product, it's fine this way for now.

**Pull request merged**